### PR TITLE
feat: add some README notes on available settings

### PR DESCRIPTION
**Pull request by cryptonemo**
Addresses #1172

### CODE REVIEW

**Comment by cryptonemo** [comment]
Review feedback/corrections for docs is most useful as a commit/update to this branch.

**Comment by cryptonemo** [comment]
> Should there be something here for the parents cache?

Sure, please add!

**Comment by porcuquine** [comment]
@dignifiedquire Can you add a short description for the parents cache, including any constraints or considerations controlling how that value should be set?

File: README.md [hunk 1]
```diff
@@ -159,34 +172,79 @@ One of the most computational expensive operations during replication (besides t
...
+Note that this setting is enabled by default.  It can be disabled by setting the value to 0.
 
-### Memory
+A related setting that can also be tuned is the SDR parents cache size.  This value is defaulted to 2048 nodes, which is the equivalent of 384KiB of resident memory (where each cached node is (6 * 32) = 192 bytes in length).  Given that the cache is now located on disk, it is memory mapped when accessed in window sizes related to this variable.  This default was chosen to minimize memory while still allowing efficient access to the cache.  If you would like to experiment with alternate sizes, you can modify the environment variable
```
Comment (cryptonemo) [inline README.md:181]: @dignifiedquire Can you check that my sizes above are sane?  I pulled those out from looking at the code, but there was a comment you made elsewhere that indicated each cached node was 56 bytes (i.e. using 112KiB RAM), so I'm looking for clarification/corrections here.

File: storage-proofs/porep/src/stacked/vanilla/cache.rs [hunk 2]
```diff
@@ -17,6 +18,7 @@ use storage_proofs_core::{
 use super::graph::{StackedGraph, DEGREE};
 
 /// Path in which to store the parents caches.
+pub const PARENT_CACHE_ENV_VAR: &str = "FIL_PROOFS_PARENT_CACHE";
```
Comment (dignifiedquire) [inline storage-proofs/porep/src/stacked/vanilla/cache.rs:21]: why not use `settings.rs` for this?

Comment (porcuquine) [reply storage-proofs/porep/src/stacked/vanilla/cache.rs:21]: Yes, please.

Comment (cryptonemo) [reply storage-proofs/porep/src/stacked/vanilla/cache.rs:21]: I went with the existing approach taken by the PARAMETER_CACHE

Comment (cryptonemo) [reply storage-proofs/porep/src/stacked/vanilla/cache.rs:21]: But can move them both to settings

File: README.md (hunk 1 above)
Comment (cryptonemo) [reply README.md]: Thanks for clarifying, it's been updated.

**Pull request merged**